crates/store/re_video/examples/frames.rs

```diff
@@ -37,11 +37,12 @@ fn main() {
     let mut decoder = re_video::decode::new_decoder(video_path.to_string(), &video)
         .expect("Failed to create decoder");
 
-    write_video_frames(&video, decoder.as_mut(), &output_dir);
+    write_video_frames(&video, &video_data, decoder.as_mut(), &output_dir);
 }
 
 fn write_video_frames(
     video: &re_video::VideoData,
+    video_data: &[u8],
```

Comment:
the combination of `re_video::VideoData` and a `video_data` parameter that isn't of that type is massively confusing

Reply:
maybe `video_blob`?

Reply:
👍 works for me, that gets it closer to the blob :)